weaviate/collection/classes.py

```diff
@@ -324,9 +324,30 @@ class _MultiTenancyConfig:
     enabled: bool
 
 
+@dataclass
+class _ReferenceDataType:
```

Comment:
_ReferenceDataType vs ReferenceDataType might be quite confusing

Reply:
suggestion:
Move all the return classes into an extra file/package to avoid confusion. How about return_classes.py?

Reply:
Absolutely! The intent here was to have `ReferenceDataType` exported by `weaviate.weaviate_classes` but not `_ReferenceDataType`. I'll open another PR for rearranging of the classes as `collection.classes.py` and `collection.grpc.py` are getting top big now

Reply:
sounds good! Then I'll resolve this one